api/admin/coordinator.py

```diff
@@ -154,6 +154,19 @@ class CoordinatorAdmin(VersionAdmin):
     ]
     action_form = GroupActionForm
 
+    list_display_links = (
+        "get_personal_info_id",
+        "get_personal_info_full_name",
+        "get_additional_skills_comment",
+        "get_project_status",
+        "get_situational_status",
+        "get_status_since",
+        "active_groups_count",
+        "get_communication_language_mode",
+        "mentor",
+        "get_comment",
+    )
```

Comment:
I understand the solution, but the idea was to make the whole row clickable, not text-containing cells in it. 

It's a more difficult task, I know. It probably should even include the highlighting of the row the mouse cursor is hovering over.

If it's not doable at all, we should only leave ID and full name clickable, otherwise it would look confusing

Reply:
I got it. I'll look how could be realize it. :)